app/_data/tables/plugin_index.yml

```diff
@@ -0,0 +1,558 @@
+
+    - name: OAuth 2 Introspection
+      free: No
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/oauth2-introspection
+
+    - name: OpenID Connect
+      free: No
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/openid-connect
+
+    - name: Session
+      free: Yes
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/session
+
+    - name: Vault Auth
+      free: No
+      plus: No
+      enterprise: Yes
+      topologies: Self-managed classic, DB-less, and hybrid
+      notes: --
+      url: /hub/kong-inc/vault-auth
+
+- category:
+  name: Security
+  plugins:
+    - name: Acme
+      free: Yes
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/acme
+
+    - name: Bot Detection
+      free: Yes
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/bot-detection
+
+    - name: CORS
+      free: Yes
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/cors
+
+    - name: IP Restriction
+      free: Yes
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/ip-restriction
+
+    - name: OPA
+      free: No
+      plus: Yes
+      enterprise: Yes
+      topologies: Self-managed classic, DB-less, and hybrid
+      notes: |
+        Available starting with Kong Gateway 2.4.
+        <br><br>
+        Konnect Cloud does not yet support 2.4.
+      url: /hub/kong-inc/opa
+
+- category:
+  name: Traffic Control
+  plugins:
+    - name: ACL
+      free: Yes
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/acl
+
+    - name: Canary
+      free: No
+      plus: No
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/canary
+
+    - name: Forward Proxy
+      free: No
+      plus: No
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/forward-proxy
+
+    - name: GraphQL Proxy Caching Advanced
+      free: No
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: --
+      url: /hub/kong-inc/graphql-proxy-cache-advanced
+
+    - name: GraphQL Rate Limiting Advanced
+      free: No
+      plus: Yes
+      enterprise: Yes
+      topologies: All
+      notes: |
+        Limitation in self-hosted hybrid and DB-less: the cluster policy is not
+        supported in DB-less and hybrid modes.
+        <br><br>
+        For DB-less and hybrid, use <code>local</code> for rate limiting individual data
+        planes, or use <code>redis</code> for cluster-wide rate limiting. If you want to apply
+        rate limiting globally over control and data planes in a hybrid deployment,
+        use <code>redis</code>.
```

Comment:
I'm shortening this to:
The <code>cluster</code> policy is not supported in DB-less and hybrid modes.

or, in places where it's a strategy, strategy instead of policy... They can click into the plugin to find out what is supported.